### PR TITLE
Fix & Add tags titles.

### PR DESCRIPTION
**Pull request by OryWareKyrie**
Addition of Simulcast and Uncut to the track titles. in case simulcast uses ```--simul``` by default Uncut is used.
```Video
ID                          : 1
Format                      : AVC
Format/Info                 : Advanced Video Codec
Format profile              : High@L4
Format settings             : CABAC / 4 Ref Frames
Format settings, CABAC      : Yes
Format settings, Reference  : 4 frames
Codec ID                    : V_MPEG4/ISO/AVC
Duration                    : 25 min 12 s
Nominal bit rate            : 8 000 kb/s
Width                       : 1 920 pixels
Height                      : 1 080 pixels
Display aspect ratio        : 16:9
Frame rate mode             : Constant
Frame rate                  : 23.976 (24000/1001) FPS
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Progressive
Bits/(Pixel*Frame)          : 0.161
Title                       : 日本語 [Simulcast]
Writing library             : x264 core 157 r2948 dada181
Encoding settings           : cabac=1 / ref=1 / deblock=1:0:0 / analyse=0x3:0x113 / me=hex / subme=7 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=24 / lookahead_threads=4 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=2 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=2 / keyint=48 / keyint_min=4 / scenecut=40 / intra_refresh=0 / rc_lookahead=40 / rc=cbr / mbtree=1 / bitrate=8000 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=8000 / vbv_bufsize=24000 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:1.00
Default                     : Yes
Forced                      : No

Audio #1
ID                          : 2
Format                      : AAC LC
Format/Info                 : Advanced Audio Codec Low Complexity
Codec ID                    : A_AAC-2
Duration                    : 25 min 12 s
Channel(s)                  : 2 channels
Channel layout              : L R
Sampling rate               : 48.0 kHz
Frame rate                  : 46.875 FPS (1024 SPF)
Compression mode            : Lossy
Delay relative to video     : -43 ms
Title                       : 日本語 [Simulcast]
Language                    : Japanese
Default                     : Yes
Forced                      : No

Audio #2
ID                          : 3
Format                      : AAC LC
Format/Info                 : Advanced Audio Codec Low Complexity
Codec ID                    : A_AAC-2
Duration                    : 25 min 12 s
Channel(s)                  : 2 channels
Channel layout              : L R
Sampling rate               : 48.0 kHz
Frame rate                  : 46.875 FPS (1024 SPF)
Compression mode            : Lossy
Delay relative to video     : -43 ms
Title                       : Português (Brasil) [Simulcast]
Language                    : Portuguese
Default                     : Yes
Forced                      : No

Audio #3
ID                          : 4
Format                      : AAC LC
Format/Info                 : Advanced Audio Codec Low Complexity
Codec ID                    : A_AAC-2
Duration                    : 25 min 12 s
Channel(s)                  : 2 channels
Channel layout              : L R
Sampling rate               : 48.0 kHz
Frame rate                  : 46.875 FPS (1024 SPF)
Compression mode            : Lossy
Delay relative to video     : -43 ms
Title                       : Español (Latinoamericano) [Simulcast]
Language                    : Spanish
Default                     : Yes
Forced                      : No

Text #1
ID                          : 5
Format                      : ASS
Codec ID                    : S_TEXT/ASS
Codec ID/Info               : Advanced Sub Station Alpha
Compression mode            : Lossless
Title                       : English (United State) [Simulcast]
Language                    : English
Default                     : Yes
Forced                      : No

Text #2
ID                          : 6
Format                      : ASS
Codec ID                    : S_TEXT/ASS
Codec ID/Info               : Advanced Sub Station Alpha
Compression mode            : Lossless
Title                       : Español (Latinoamericano) [Simulcast]
Language                    : Spanish
Default                     : Yes
Forced                      : No

Text #3
ID                          : 7
Format                      : ASS
Codec ID                    : S_TEXT/ASS
Codec ID/Info               : Advanced Sub Station Alpha
Compression mode            : Lossless
Title                       : Português (Brasil) [Simulcast]
Language                    : Portuguese
Default                     : Yes
Forced                      : No
```
```[INFO] Selected Episodes: 0001

[INFO] Kaguya-sama: Love is War - S1E1 - I Will Make You Invite Me to a Movie / Kaguya Wants to Be Stopped / Kaguya Wants It
[INFO] Available streams (Non-Encrypted):
[#801045] Japanese [Simulcast] (selected) (using 'English', 'Spanish (Latin Am)', 'Portuguese (Brazil)' for subtitles)
[#1699379] Portuguese (Brazil) [Simulcast] (selected)
[#1699247] Spanish (Latin Am) [Simulcast] (selected)
```
Valeu pela ajuda vovó @alfablac :smiley_cat:

### CODE REVIEW

**Review by izu-co — CHANGES_REQUESTED** [review]
Hello there. 

First, many thanks for going to the trouble of programming an extension to this tool.

There are some things I have to request to be changed, before I can merge this PR, however.

 - Why did you delete the `package-lock.json` file?'
 - I think your code base is not up-to-date (haven't checked, just a feeling)?
 - Lastly, there are a lot of style changes in the files. Is there a particular reason for that?

**Comment by OryWareKyrie** [comment]
> Hello there.
> 
> First, many thanks for going to the trouble of programming an extension to this tool.
> 
> There are some things I have to request to be changed, before I can merge this PR, however.
> 
> * Why did you delete the `package-lock.json` file?'
> * I think your code base is not up-to-date (haven't checked, just a feeling)?
> * Lastly, there are a lot of style changes in the files. Is there a particular reason for that?
> 
> There are some conflicts by the way that `must` be resolved before I can even merge it :).

bro I honestly don't know what happened. even to prevent this, I took the most recent version of the code, anidl, and made the changes to it. I think when formatting the files with eslint I have deleted something. I formatted all the documents.

**Comment by izu-co** [comment]
izu [this](https://github.com/anidl/funimation-downloader-nx/blob/master/config/cli-defaults.yml#L1) line has some function in the tool? 🤔

Not anymore

File: config/cli-defaults.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-releaseGroup: Funimation
 videoLayer: 7
 nServer: 1
 mp4mux: false
-muxSubs: false
+muxSubs: true
```
Comment (izu-co) [inline config/cli-defaults.yml:4]: You can remove this line since it isn't used anymore

File: modules/module.getdata.js [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
+    if (options.auth) {
         gOptions.method = 'POST';
-        gOptions.body = new FormData();
-        gOptions.body.append('username', options.auth.user);
-        gOptions.body.append('password', options.auth.pass);
+        gOptions.headers['Content-Type'] = 'application/x-www-form-urlencoded; charset=UTF-8';
+        gOptions.headers['Origin'] = 'https://www.funimation.com';
+        gOptions.headers['Accept'] = 'application/json, text/javascript, */*; q=0.01';
+        gOptions.headers['Accept-Encoding'] = 'gzip, deflate, br';
+        gOptions.headers['User-Agent'] = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0';
```
Comment (izu-co) [inline modules/module.getdata.js:47]: These changes are already in the newest version, are they not?

**Review by izu-co — CHANGES_REQUESTED** [review]

**Comment by izu-co** [comment]
@OryWareKyrie I think it would be best if you apply your changes to a fork of this repo and not my fork. 

I think it would save a lot of trouble.

**Comment by OryWareKyrie** [comment]
> @OryWareKyrie I think it would be best if you apply your changes to a fork of this repo and not my fork.
> 
> I think it would save a lot of trouble.

I think I will do this even izu :smile_cat:

**Pull request closed without merging**